src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__item:last-child {
+    margin: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    color: $linkTextColor;
+    position: relative;
+  }
+
+  &__link:hover,
+  .is-active {
+    color: $active;
+  }
+
+  & .is-active::after {
```

Comment:
The `.is-active` class should be nested inside the `&__link` block for the same reasons as mentioned above. It should be `&__link.is-active::after` to correctly represent it as a modifier of the `&__link` element.